invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -98,7 +98,7 @@ export default class LoanMetadata extends Component {
       },
       {
         name: 'Delivery',
-        value: data.metadata.delivery.method,
+        value: data.metadata.delivery ? data.metadata.delivery.method : null,
```

Comment:
you might have conflict with Karolina's PR

Reply:
you will :)